sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/Cast.scala

```diff
@@ -959,15 +972,25 @@ case class Cast(child: Expression, dataType: DataType, timeZoneId: Option[String
       evPrim: ExprValue, evNull: ExprValue, canNullSafeCast: Boolean): Block = {
     if (canNullSafeCast) {
       code"""
-         |$d.changePrecision(${decimalType.precision}, ${decimalType.scale});
+         |$d.changePrecision(
+         |  ${decimalType.precision}, ${decimalType.scale});
```

Comment:
nit: write this in a single line? `         |$d.changePrecision(${decimalType.precision}, ${decimalType.scale});`